wp-content/themes/sceraTheatre/footer.php

```diff
@@ -1,17 +1,10 @@
 <div class="sponsors">
-	<h2>Thanks to our Sponsors</h2>
-	<div class="sponsorImages">
-		<div class="imageWrapper">
-		
-		</div>
-
-		<div class="imageWrapper">
-		
-		</div>
+	<div class="sponsorTitle">Thanks to our </br> Sponsors</h2> </div>
```

Comment:
Just change .sponsorTitle to "title blue". Those styles are already there. I also don't see an opening <h2> tag anywhere. Also, we probably don't want to force a line break. Just let it break on its own.